### DISCUSSION TITLE
Welcome to OPENCV Projects Discussions!

### DISCUSSION THREAD

**Discussion by Chetan1604** [Chetan1604/Computer_Vision_Algorithms#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by akshitagupta15june** [comment, marked as answer]
I would like to add opencv projects in this repo

**Comment by Chetan1604** [comment]
You can add them

**Comment by Chetan1604** [comment]
Basic face detection needed first

**Comment by Chetan1604** [comment]
Vehicle number plate detection

**Comment by akshitagupta15june** [comment]
vehicle classification
